CONTRIBUTING.md

```diff
@@ -31,14 +30,31 @@ here: https://guides.rubygems.org/contributing/
 ## Getting Started
 
     $ rake setup
-    $ rake test
 
-> Optional you can configure git hooks with: rake git_hooks
+> Optionally you can configure git hooks with: rake git_hooks
 
 To run commands like `gem install` from the repo:
 
     $ ruby -Ilib bin/gem install
 
+### Running Tests
+
+To run the entire test suite you can use: 
+
+    $ rake test
+
+To run an individual test file located for example in `test/rubygems/test_deprecate.rb` you can use: 
+
+    $ ruby -Ilib:test test/rubygems/test_deprecate.rb 
+    
+And to run an individual test method named `test_default` within a test file, you can use: 
+
+    $ ruby -Ilib:test test/rubygems/test_deprecate.rb -n /test_default/ 
+
+You can even run a test against your currently installed RubyGems version instead of the repository itself if you want to check whether the test would be failing on it as well:
```

Comment:
I'm not sure how safe this is. I'd say it shouldn't perform any destructive actions on your system install, but I'm not fully sure. Maybe keep that trick "a secret" at least for now?

Reply:
Ok, removed. :-)